tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -3326,3 +3378,30 @@ def test_ratelimit(self):
             for i in range(5):
                 self.get_success_response()
             self.get_error_response(status_code=status.HTTP_429_TOO_MANY_REQUESTS)
+
+    def test_bulk_delete_performance_issues(self):
+        groups = []
+        for i in range(10, 41):
+            groups.append(
+                self.create_group(
+                    project=self.project,
+                    status=GroupStatus.RESOLVED,
+                )
+            )
+
+        hashes = []
+        for group in groups:
+            hash = uuid4().hex
+            hashes.append(hash)
+            group.update(type=GroupType.PERFORMANCE_SLOW_SPAN.value)
```

Comment:
I think you can just add `type=GroupType.PERFORMANCE_SLOW_SPAN.value` as a kwarg to `self.create_group` and it should just be added to the newly created group. No need for a second loop.